pkg/operation/botanist/etcd.go

```diff
@@ -191,6 +192,63 @@ func (b *Botanist) scaleETCD(ctx context.Context, replicas int32) error {
 	return b.Shoot.Components.ControlPlane.EtcdEvents.Scale(ctx, replicas)
 }
 
+func (b *Botanist) deployOrRestoreMainEtcd(ctx context.Context) error {
+	isRestoreRequired, err := b.isRestorationOfMultiNodeMainEtcdRequired(ctx)
+	if err != nil {
+		return err
+	}
+
+	if isRestoreRequired {
+		return b.restoreMultiNodeMainEtcd(ctx)
```

Comment:
Good question, actually I hadn't noticed that the various replica settings on the HVPA resource also depend on the current replica count and are not modified by the `Scale()` function. However, I did not see any problems with that. 
Maybe in the `restoreMultiNodeMainEtcd()` function I could set the replicas to 3 and call `Deploy()` directly, instead of relying on scale

Reply:
To resolve this, we decided that it's be better to simply have the `Scale()` function also update the `HVPA` so that the replicas specified in its `spec.hpa.template.spec.maxReplicas` and `spec.hpa.template.spec.minReplicas` are kept consistent with the replicas of the `Etcd` resource , even though its `spec.hpa.deploy` field is `false`.